playwright/tests/girftCards.spec.ts

```diff
@@ -100,3 +104,29 @@ test("TC: SALEOR_111 Bulk delete gift cards @e2e @gift", async () => {
     ),
   ).toEqual([]);
 });
+test("TC: SALEOR_112 Set gift card balance @e2e @gift", async () => {
+  await giftCardsPage.gotoExistingGiftCardView(GIFT_CARDS.giftCardToBeEdited.id);
+  await giftCardsPage.clickSetBalance();
+  await giftCardsPage.setGiftCardsBalanceDialog.setBalance("34")
+  await giftCardsPage.expectSuccessBanner();
+});
+test("TC: SALEOR_113 Export gift card codes in XLSX file @e2e @gift", async () => {
+  await giftCardsPage.gotoGiftCardsListView();
+  await giftCardsPage.clickShowMoreMenu();
+  await giftCardsPage.clickExportGiftCards();
+  await giftCardsPage.exportGiftCardsDialog.exportGiftCardCodes("XLSX");
+  await mailpitService.checkDoesUserReceivedExportedData(
+    process.env.E2E_USER_NAME!,
+    "Your exported gift cards data is ready",
+  );
+});
+test("TC: SALEOR_114 Export gift card codes in CSV file @e2e @gift", async () => {
+  await giftCardsPage.gotoGiftCardsListView();
+  await giftCardsPage.clickShowMoreMenu();
+  await giftCardsPage.clickExportGiftCards();
+  await giftCardsPage.exportGiftCardsDialog.exportGiftCardCodes("CSV");
+  await mailpitService.checkDoesUserReceivedExportedData(
+    process.env.E2E_USER_NAME!,
+    "Your exported gift cards data is ready",
+  );
```

Comment:
It made me think that if SALEOR_113 passes then SALEOR_114 will pass even if mail won't be sent - because SALEOR_114 will find a message from SALEOR_113. Right?

Reply:
If you tested it and it works then let's not overcomplicate it :)